File: Python/Assignments/chartype.py
inp_input = input("Enter a character: ")


if 32 < ord(inp_input) < 48 or 57 < ord(inp_input) < 65 or 91 < ord(inp_input) < 97 or 122 < ord(inp_input) < 127:
  print(inp_input, "is a non-alphanumeric character.")
elif 47 < ord(inp_input) < 58:
  print(inp_input, "is a digit.")
elif 64 < ord(inp_input) < 91:
  print(inp_input, "is an uppercase letter.")
elif 96 < ord(inp_input) < 123:
  print(inp_input, "is a lowercase letter.")

File: C++/Assignments/chartype.cpp
#include <iostream>
using namespace std;


int main ()
{
char userCh;
cout << "Please enter a character:"<<endl;
cin >> userCh;

if(userCh >= 'a' && userCh <= 'z')
cout<<userCh << " is a lower case letter."<<endl;
else if(userCh >= 'A' && userCh <= 'Z')
cout<<userCh << " is an uppercase letter"<<endl;
else if(userCh >= '0' && userCh <= '9')
cout<<userCh << " is a digit"<<endl;
else
cout<<userCh << " is a non-alphanumeric character"<<endl;

return 0;
}
